Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -122,6 +122,12 @@ def tests
       unless OS.mac?
         bundle_args << "--tag" << "~needs_macos" << "--tag" << "~cask"
         files = files.reject { |p| p =~ %r{^test/(os/mac|cask)(/.*|_spec\.rb)$} }
+
+        if HOMEBREW_PATCHELF_RB || HOMEBREW_READELF_RB
+          # don't overwrite the gem environment setup for bundler
+          # :: PATCHELF_VERSION
```

Comment:
What's the purpose of these `:: PATCHELF_VERSION` comments? Are they for grepping? We don't do that elsewhere I don't believe, so let's remove them.

Reply:
they are for my personal help, in future if the patchelf gem version changes,
i'm making it easier to change in places where the patchelf version is hardcoded.

In short, i dont wish to use a  a global variable for holding the version number, there by using a comment as proxy.

Reply:
> In short, i dont wish to use a a global variable for holding the version number, there by using a comment as proxy.

I think a global variable would be better.

Reply:
using a global variable now.